projects/plugins/crm/sass/_semantic-ui.scss

```diff
@@ -1,5 +1,5 @@
-@import 'semantic-ui/globals/all';
-@import 'semantic-ui/elements/all';
-@import 'semantic-ui/collections/all';
-@import 'semantic-ui/views/all';
-@import 'semantic-ui/modules/all';
\ No newline at end of file
+@forward 'semantic-ui/globals/all';
+@forward 'semantic-ui/elements/all';
+@forward 'semantic-ui/collections/all';
+@forward 'semantic-ui/views/all';
+@forward 'semantic-ui/modules/all';
```

Comment:
Actually, this file should just be removed. I removed refs to it in #43778, but accidentally didn't delete it.